script.js

```diff
@@ -1,4 +1,90 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var GAME_STATE_DICE_ROLL = 'GAME_STATE_DICE_ROLL';
+var GAME_STATE_CHOOSE_DICE_ORDER = 'GAME_STATE_CHOOSE_DICE_ORDER';
+var GAME_STATE_COMPARE_SCORE = 'GAME_STATE_COMPARE_SCORE';
+var gameState = GAME_STATE_DICE_ROLL;
+
+var currentPlayerRolls = [];
+var currentPlayer = 1;
+var allPlayersScore = [];
+
+var rollDice = function () {;
+  var randomDecimal = Math.random() * 6;
+  var resultInteger = Math.floor(randomDecimal);
+  var diceNumber = resultInteger + 1;
+  console.log ('rollDice', diceNumber);
+  return diceNumber;
+}
+
+var rollDiceForPlayer = function (){;
```

Comment:
Nice usage of helper functions, I like how you use the loop to indicate that you're doing two rolls! Is this a possible hint that you might want to do more than two dice later on?